File: environments_harbor/gradle_build_cache_configuration_medium/environment/java-app/src/test/java/com/example/CalculatorTest.java
package com.example;

import org.junit.Test;
import org.junit.Before;
import org.junit.Assert;

public class CalculatorTest {
    
    private Calculator calculator;
    
    @Before
    public void setUp() {
        calculator = new Calculator();
    }
    
    @Test
    public void testAddition() {
        int result = calculator.add(5, 3);
        Assert.assertEquals(8, result);
    }
    
    @Test
    public void testSubtraction() {
        int result = calculator.subtract(10, 4);
        Assert.assertEquals(6, result);
    }
    
    @Test
    public void testMultiplication() {
        int result = calculator.multiply(6, 7);
        Assert.assertEquals(42, result);
    }
    
    @Test
    public void testDivision() {
        int result = calculator.divide(20, 4);
        Assert.assertEquals(5, result);
    }
    
    @Test
    public void testAdditionWithNegativeNumbers() {
        int result = calculator.add(-5, 3);
        Assert.assertEquals(-2, result);
    }
}

File: environments_harbor/phpunit_test_suite_medium/environment/workspace/calculator-service/tests/CalculatorTest.php
<?php

namespace Tests;

use PHPUnit\Framework\TestCase;
use App\Calculator;

class CalculatorTest extends TestCase
{
    private $calculator;

    protected function setUp(): void
    {
        $this->calculator = new Calculator();
    }

    public function testAddition()
    {
        $result = $this->calculator->add(5, 3);
        $this->assertEquals(8, $result);
        
        $result = $this->calculator->add(-2, 7);
        $this->assertEquals(5, $result);
        
        $result = $this->calculator->add(0, 0);
        $this->assertEquals(0, $result);
    }

    public function testSubtraction()
    {
        $result = $this->calculator->subtract(10, 4);
        $this->assertEquals(6, $result);
        
        $result = $this->calculator->subtract(5, 10);
        $this->assertEquals(-5, $result);
    }

    public function testMultiplication()
    {
        $result = $this->calculator->multiply(4, 5);
        $this->assertEquals(20, $result);
        
        $result = $this->calculator->multiply(-3, 6);
        $this->assertEquals(-18, $result);
        
        $result = $this->calculator->multiply(0, 100);
        $this->assertEquals(0, $result);
    }
}